WhaleSpotting/Controllers/SightingController.cs

```diff
@@ -32,9 +33,13 @@ public IActionResult Index()
     }
     
     [HttpPost("")]
-    public IActionResult NewSighting([FromForm] Sighting newSighting)
+    public IActionResult NewSighting([FromForm] NewSightingFormViewModel newSightingViewModel)
     {
-        _context.Sightings?.Add(newSighting);
+        var speciesId = newSightingViewModel.SpeciesId;
+        Species species = _context.Species!
+            .Where(speciesCheck => speciesCheck.Id == speciesId).First();
```

Comment:
Minor, but splitting this over a couple of lines can make it slightly easier to read:
```suggestion
            .Where(speciesCheck => speciesCheck.Id == speciesId)
            .First();
```